db/schema.rb

```diff
@@ -46,6 +46,7 @@
     t.datetime "remember_created_at"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.string "username"
```

Comment:
are there any validations for the username? Like minimum or maximum number of characters? Whether it is unique?

Reply:
Thanks Mina! We added a card for username validations to our trello board.